competencies/roles/_TechWideGeneral.md

```diff
@@ -122,3 +122,76 @@
+and variants of their headspace (users thinking about new home or about financing, users thinking about existing home)
+
+
+##### Technology
+
+Aware of the most relevant trends and levers in technology industry, in order to identify good patterns
+and contribute to technical experiments
+- Role of managed services in reducing effort of a team and increasing a team's ability to focus on their 
+core offering
+- Role of cloud technologies in elasticity and on demand availability
+- DevOps role in enabling frequent small (lower risk) releases and encouraging maintainable/operable software
+- Trends in languages and tools and the role in fast development and maintainable development
+
+Contextualizes tech decisions such as
+
+- Understand why we don't always build and operate our selves
+- Understand that some technical choices are hard to change and important
+- Understand that branching, release, testing and other strategies that affect the developer experience 
+can affect quality and effectiveness (and how that can improve user experience, business outcomes, and recruiting) 
+
+#### Examples of Proficient
+##### Real Estate
+Aware of current or nuanced trends Canadian real estate and actively uses those in the broader work. Examples of the type of information:
+- The idea of balanced vs sellers vs buyers market (and the idea of uncertainty) to inform what work might be a priority
+  - e.g. confidence tools vs move fast on dream home vs affordability
+- The effect of interest rates on affordability and how to think about overall business risk of slowdown 
+(comfort with taking decreasing business risk, awareness (resilience) that the cycle will pass  )
+- The effect of home value and of uncertainty on our business model, and ability to contribute ideas on how to take
+benefit or avoid negative impact.
+- New opportunities created by changing dynamics (e.g. Properly has broader capabilities and context than realtors or banks)
+
+This experience and knowledge on the real estate side for a tech team member helps them contribute to prioritization and approach with
+product and design partners
+
+##### Technology
+
+Deeply aware of and reasoning about current or more nuanced movement in the technology space, examples
+- Platform ownership and what that might mean (e.g. Looker owned by google)
+- Team trends around practices (e.g. trends in testing and quality, security and privacy practices - e.g. Apple positioning)
+- Discerning about tradeoffs with vendors and effect of their business model on our scaling (how to think about Algolia vs Elastic Search models)
+
+This experience and connectedness to technology trends enables a team member to contribute more effectively
+in planning and in system design activities by identifying future needs and the technology and vendor trends 
+that are likely to be tailwinds or headwinds to our technical delivery. 
+
+#### Examples of Authority
+##### Real Estate
+
```

Comment:
Would we consider basic understanding of CM, and the various components that go into CM1, 2, 3, 4, to be demonstration of proficiency or authority? I might lean towards authority for a technical contributor, just checking here.

Reply:
I think "Understands unit economics of the business" implies we'd expect basic understanding of the CM indicators at the Proficient level